backend/api/Controllers/RobotController.cs

```diff
@@ -277,6 +280,51 @@ public async Task<ActionResult<IList<VideoStream>>> GetVideoStreams([FromRoute]
         return Ok(robot.VideoStreams);
     }
 
+    /// <summary>
+    /// Get all plants associated with an active robot.
+    /// </summary>
+    /// <remarks>
+    /// <para> Retrieves the plants that have an active robot </para>
+    /// </remarks>
+    [HttpGet]
+    [Authorize(Roles = Role.User)]
+    [Route("active-plants")]
+    [ProducesResponseType(typeof(IList<EchoPlantInfo>), StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    public async Task<ActionResult<IList<EchoPlantInfo>>> GetActivePlants()
```

Comment:
Does this make sense to have in the robotService? Would it be better to have this call in the echoService and it can call the robot service to know what active robots there are in order to filter?

Reply:
Are you suggesting we put it in the echoController? I suppose we could also have it in the plantController

Reply:
I moved it to the echo controller